### DISCUSSION TITLE
Status option

### DISCUSSION THREAD

**Discussion by IndrajeethY** [tulir/whatsmeow#231] [General]
Is is possible to set a video or image as whatsapp status just like Android app does.? Also is there any similar library's in python with updated feature. And can I start a video call using this library'?

**Comment by mamur-rezeki** [comment]
Calls are not yet implemented, you can read last section of README file.

**Reply by IndrajeethY** [reply]
oh ok wt about rest of my  queriies?

**Comment by mamur-rezeki** [comment]
WhatsApp status just like a regular message protocol like `
proto.ExtendedTextMessage{} `for Text Status (some fields could be
customized for a better view).

The Media status it's also work for  ` ImageMessage ` , ` VideoMessage `.

The target jid to sent is ` status@broadcast `.

**Comment by GustavoGSA** [comment]
Hi, this work now? Lik this...

![image](https://github.com/user-attachments/assets/a9c11061-8e60-40a9-8663-736ea8f0d27d)

**Reply by ghost** [reply]
That is `PtvMessage`, just another `VideoMessage`.
